extension.js

```diff
@@ -362,7 +362,7 @@ function createSelectStatusBar() {
 function syncStatusBar(memoryStatusBarArray) {
     const diff = memoryStatusBarArray.length - statusBarArray.length;
     for (let i = 0; i < diff; ++i) {
-        let statusBar = vscode.window.createStatusBarItem("actboy168.tasks", vscode.StatusBarAlignment.Left, 50);
+        let statusBar = vscode.window.createStatusBarItem(`actboy168.tasks.task-${statusBarArray.length + i}`, vscode.StatusBarAlignment.Left, 50);
```

Comment:
```suggestion
        let statusBar = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Left, 50);
```

Reply:
Using a different id will break the previous behavior

Reply:
I am curious as to how - as the not having of a unique id is what is causing the current issue that only the last one is shown?
I don't know the API so I made the change most logical to me - which was adding the index to the id - so it's unique per item/task

Reply:
Or are you referring to your comment about the uniform disable/enable/hide functionality?
https://github.com/actboy168/vscode-tasks/issues/44#issuecomment-1582184164

Reply:
yes

Reply:
I made the requested change based on: https://github.com/actboy168/vscode-tasks/issues/44#issuecomment-1582506027